### PR TITLE
Add BRANCH_NAME env and modify default branch for effected step 

### PR DESCRIPTION
**Pull request by ImMin5**
### Description
To solving a problem that failures when triggering the Git workflow from a different branch.

- add BRANCH_NAME env
- modify default branch for effected step

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM~!

**Review by sulmoJ — APPROVED** [review]
LGTM~~

File: workflows/core/api/dispatch_release.yaml [hunk 1]
```diff
@@ -21,6 +22,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           token: ${{ secrets.PAT_TOKEN }}
+          ref: ${{ env.BRANCH_NAME }}
```
Comment (sulmoJ) [inline workflows/core/api/dispatch_release.yaml]: delete!

**Comment by ImMin5** [comment]
This PR move to https://github.com/cloudforet-io/actions/pull/43

**Pull request closed without merging**